frontend/testing/cypress/src/integration/studio/designer.js

```diff
@@ -43,24 +43,48 @@ context('Designer', () => {
 
     // Add new page and ensure updated data is loaded
     designer.getAddPageButton().click();
+
     cy.wait('@postLayoutSettings').its('response.statusCode').should('eq', 200);
     cy.wait('@getLayoutSettings').its('response.statusCode').should('eq', 200);
 
-    // Verify navigation button exists in form
-    designer.getDroppableList().findByRole('listitem', { name: texts['ux_editor.component_navigation_buttons'] }).should('be.visible');
-
     // Add an input component
     designer.getToolbarItemByText(texts['ux_editor.component_input']).trigger('dragstart');
     designer.getDroppableList().trigger('drop');
     cy.wait(500);
-    designer.getDroppableList()
+    designer
+      .getPageAccordionByName('Side1')
       .findAllByRole('listitem')
-      .then(($elements) => expect($elements.length).eq(2));
+      .then(($elements) => expect($elements.length).eq(1));
 
     // Delete components on page
     cy.deleteComponents();
```

Comment:
Testet dette lokalt hos meg og det fungerer fint. Var det her du fikk problemer? 🤔 Kunne du prøvd igjen med koden under for å se om det fungerer fint? Det burde fungere da testen er skrevet mer direkte mot det vi faktisk ønsker å teste. Her får vi testet at komponenten vi drar inn faktisk er den som dukker opp også.

```suggestion
  designer
      .getPageAccordionByName('Side1')
      .findByRole('listitem', { name: texts['ux_editor.component_input'] });
```

Reply:
Ikke der jeg hadde problemer nei, men endringen du foreslår fungerer og er nok bedre å bruke i dette tilfellet ja 😄